src/cpp/diagnostics/DiagnosticsMain.cpp

```diff
@@ -47,30 +48,44 @@ FilePath userLogPath()
    return logPath;
 }
 
-void writeLogFile(const std::string& logFileName, std::ostream& ostr)
+void writeFile(const std::string& fileName, const core::FilePath& path, std::ostream& ostr)
```

Comment:
nit: maybe `description` or `scope` instead of `fileName`?

Reply:
👍